test/integration/password_reset_test.rb

```diff
@@ -97,6 +97,29 @@ def forgot_password_with(email)
     assert page.has_content?("Sign out")
   end
 
+  test "resetting password when webauthn is enabled" do
```

Comment:
I think we also need an integration test for email confirmations controller (https://github.com/Shopify/rubygems.org/blob/3cc8bb155604c067b30d6218ee3bc7213675a50b/test/functional/email_confirmations_controller_test.rb)

Reply:
Added.